.github/workflows/pr-checks.yml

```diff
@@ -61,3 +61,19 @@ jobs:
         run: yarn install --frozen-lockfile
       - name: format-check
         run: yarn format:check
+
+  typescript:
+    name: Check TypeScript
+    runs-on: ubuntu-latest
+    needs: install-node-modules
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 14.x
```

Comment:
Should this be 12?

Reply:
Thanks for noticing - I downgraded to 12x

Reply:
Sweet, thanks!